client/containers/Pub/PubDocument/Review/ReviewerDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Button, Classes, Dialog, InputGroup } from '@blueprintjs/core';
+
+require('./reviewerDialog.scss');
+
+type Props = {
+	isOpen: boolean;
+	pubData: {
+		title: string;
+	};
+	onClose: (...args: any[]) => any;
+	createReviewDoc: () => any;
+	setReviewTitle: any;
+	reviewTitle: string;
+	reviewerName: any;
+	setReviewerName: any;
+	isUser: boolean;
+};
+
+const ReviewerDialog = (props: Props) => {
+	const {
+		isOpen,
+		onClose,
+		pubData,
+		createReviewDoc,
+		setReviewTitle,
+		reviewTitle,
+		reviewerName,
+		setReviewerName,
+		isUser,
+	} = props;
+
+	const renderPreReviewButtons = () => {
+		return (
+			<React.Fragment>
+				<Button onClick={onClose}>Cancel</Button>
+				<Button intent="primary" onClick={createReviewDoc}>
+					Create Review
+				</Button>
+			</React.Fragment>
+		);
+	};
+	const helperTextToRender = (
+		<p>
+			You are about to submit a review for <b>{pubData.title}</b>.
+		</p>
+	);
```

Comment:
Should we just move this into the JSX block below since it's fully declarative/markup?